tools/version.sh

```diff
@@ -7,11 +7,10 @@ if ! test -d "${srcdir}/.git"; then
   if test -f "${version_h_path}"; then
     while read line; do
       set -- $line
-      export $2=$(echo $3 | sed 's/"//g')
+      IFS= read "$2" <<< $(echo $3 | sed 's/"//g')
```

Comment:
Not sure I follow this particular change. Also, stylistic nit: spacing around `=`.